packages/babel-plugin-named-asset-import/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "babel-plugin-named-asset-import",
-  "version": "0.3.6",
+  "version": "0.3.7",
```

Comment:
Interesting that the versions of these packages got updated - probably doesn't matter as they didn't get published, but it's a little confusing. This is lernas built in functionality right? Perhaps when you use the cli you could enter in the same version? maybe it'd throw an error.

Reply:
Yeah this is lerna doing it - will give it a try with the same version though as it shouldn't complain if its not publishing

Reply:
How did this go?

Reply:
It still tried to do it and errored but a change in the publish script for running lerna will only now publish react-scripts